### PR TITLE
Htmlandcss

### PR DESCRIPTION
**Pull request by akoladepaul**
### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

**Comment by kenny-wealth** [comment]
Always add a description to your PRs

File: submissions/PaulAkolade/html-css-task/css/index.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
+/*
```
Comment (kenny-wealth) [inline submissions/PaulAkolade/html-css-task/css/index.css:1]: This PR seems to duplicate another one, please close  one of them

**Review by kenny-wealth — CHANGES_REQUESTED** [review]
This PR seems to duplicate another one, please close  one of them

**Comment by JohnItoo** [comment]
@akoladepaul I did not notice this duplicate and when your other Pull Request from your master branch got merged this got merged too because you merged this into your master branch. Please next time try to follow the proper process so your work can be tracked properly.
Cheers and Well done.

**Pull request merged**